### PR TITLE
Added dockerfile

### PR DESCRIPTION
**Pull request by chetan253**

### CODE REVIEW

**Comment by chetan253** [comment]
Can one of the admins verify this patch?

**Comment by chetan253** [comment]
closes #104

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:latest
```
Comment (bhushanmalgaonkar) [inline Dockerfile]: Specify exact version instead of latest.

**Review by bhushanmalgaonkar — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (chetan253) [reply Dockerfile]: i'll fix it in next commit

Comment (chetan253) [reply Dockerfile]: fixed in 727e6d8bef5c9ee39bc4b996a51718e87a6d434a

**Pull request merged**